examples/alchemy-daapp/README.md

```diff
@@ -41,7 +41,7 @@ export const serverConfigs: Record<number, ServerConfiguration> = {
 ```
 ### **🗒️ Notes:** for `nftContractAddress` and `lightAccountFactoryAddress` 
 - There are already contract addresses deployed for them across [various chains here](https://github.com/alchemyplatform/aa-sdk/blob/main/examples/alchemy-daapp/src/configs/clientConfigs.ts).
-- We used Alchemy's [lightAccountFactory here](https://github.com/alchemyplatform/light-account).
+- We used [Light Account Factory here](https://github.com/alchemyplatform/light-account).
```

Comment:
let's brand this since it's in the example app
```suggestion
- We used the Alchemy [Light Account Factory here](https://github.com/alchemyplatform/light-account).
```